### PR TITLE
Bug #202145 : block the documentation page

### PR DESCRIPTION
**Pull request by swapnilphalke03**
https://tracker.tekdi.net/issues/202145

### CODE REVIEW

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 1]
```diff
@@ -97,30 +114,74 @@ const Docschecklist = () => {
...
+      {msgshow ? (
+        <Alert status="warning" alignItems={"start"} mb="3" mt="4">
+          <HStack alignItems="center" space="2" color>
+            <Alert.Icon />
+            <BodyMedium>{t("PAGE_NOT_ACCESSABLE")}</BodyMedium>
+          </HStack>
+        </Alert>
+      ) : (
+        <VStack width={"90%"} margin={"auto"} mt={3}>
+          <Text fontSize="sm" mt={3} bold color="textMaroonColor.900">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:128]: use theme typo component.

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 2]
```diff
@@ -97,30 +114,74 @@ const Docschecklist = () => {
...
-            _selectedItem={{
-              bg: "teal.600",
-              endIcon: <CheckIcon size="5" />,
-            }}
-            mt={1}
-            onValueChange={(itemValue) =>
-              setStatus({ ...status, jan_adhar: itemValue })
-            }
+          <HStack mt={8} alignItems={"center"} justifyContent={"space-between"}>
+            <Text fontSize="sm" color="textMaroonColor.400">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:132]: use theme typo component.

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 3]
```diff
@@ -97,30 +114,74 @@ const Docschecklist = () => {
...
+
+          <HStack
+            mt={8}
+            space="2"
+            alignItems={"center"}
+            justifyContent={"space-between"}
           >
-            {Array.isArray(selectData) &&
-              selectData.map((item, i) => {
+            <Text fontSize="sm" color="textMaroonColor.400">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:167]: Use theme typo component.

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 4]
```diff
@@ -129,392 +190,358 @@ const Docschecklist = () => {
...
-                <Select.Item
-                  key={i}
-                  label={`${t(item.title)}`}
-                  value={item.value}
-                />
-              );
-            })}
-          </Select>
-        </HStack>
+            <Text fontSize="sm" color="textMaroonColor.400">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:202]: use theme typo component.

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

**Review by sagarkoshti1990 — APPROVED** [review]

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 5]
```diff
@@ -20,17 +20,34 @@ import {
...
   const [lang, setLang] = React.useState(localStorage.getItem("lang"));
   const { id } = useParams();
   const [selectData, setselectData] = useState([]);
   const [status, setStatus] = useState({});
   const [checkList, setCheckList] = useState(false);
   const [buttonPress, setButtonPress] = useState(false);
   const [benificiary, setBenificiary] = React.useState({});
+  const [msgshow, setmsgshow] = React.useState(false);
+
+  const DocumentStatus = (docStatus) => {
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:33]: how can we avoid duplicate code?

File: lib/common-lib/src/components/helper.js [hunk 6]
```diff
@@ -649,3 +649,16 @@ export const setQueryParameters = (data) => {
     window.location.origin + window.location.pathname + jsonToQueryString(data)
   window.history.pushState('', '', pageUrl)
 }
+
+export const DocumentStatus = (docStatus) => {
```
Comment (manojLondhe) [inline lib/common-lib/src/components/helper.js:653]: can we rename this as `getBeneficaryDocumentationStatus`

File: lib/common-lib/src/index.tsx [hunk 7]
```diff
@@ -32,6 +32,7 @@ import * as eventService from './services/EventService'
 import * as aadhaarService from './services/aadhaarService'
 import Alert from './components/Alert'
 import GetEnumValue from './components/GetEnumValue'
+import { DocumentStatus } from './components/helper'
```
Comment (manojLondhe) [inline lib/common-lib/src/index.tsx:35]: can we import where it is needed? @sagarkoshti1990

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: lib/common-lib/src/index.tsx (hunk 7 above)
Comment (sagarkoshti1990) [reply lib/common-lib/src/index.tsx:35]: This is not needed.

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**